src/App.tsx

```diff
@@ -58,6 +59,7 @@ export function App() {
         <Route path="/solve" element={<SolvePage />} />
         <Route path="/login" element={<LoginPage />} />
         <Route path="/register" element={<RegisterPage />} />
+        <Route path="/profile" element={<ProfilePage />} />
```

Comment:
Maybe in a future iteration, we can make this use a URL parameter with the username so the link is shareable. For example, ofast.io/profile/blakeg